e2e/cypress/integration/modals/quick_switcher_spec.js

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @modals
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Quick switcher', () => {
+    let testTeam;
+    let testUser;
+    let firstUser;
+    let secondUser;
+    let thirdUser;
+
+    before(() => {
+        // # create three users for testing
+        cy.apiInitSetup().then(({user, team}) => {
+            testUser = user;
+            testTeam = team;
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+        cy.apiCreateUser({prefix: 'az1'}).then(({user: user1}) => {
+            firstUser = user1;
+        });
+
+        cy.apiCreateUser({prefix: 'az2'}).then(({user: user1}) => {
+            secondUser = user1;
+        });
+
+        cy.apiCreateUser({prefix: 'az3'}).then(({user: user1}) => {
+            thirdUser = user1;
+        });
+    });
+
+    it('MM-T3447_1 Should add recent user on top of results', () => {
+        cy.apiLogout();
+
+        // # Login as test user
+        cy.apiLogin(testUser);
+
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Type either cmd+K / ctrl+K depending on OS
+        cy.get('#post_textbox').cmdOrCtrlShortcut('K');
```

Comment:
Got failing test here both in CI and local. Passed consistently by,
``cy.focused().type(testChannel.display_name).wait(TIMEOUTS.HALF_SEC).type('{enter}');``

Note: `testChannel` is the channel from apiInitSetup.

Reply:
I guess it needed the wait.

Reply:
Thanks for updating! LGTM.